packages/bits/src/lib/wizard-v2/wizard-horizontal/wizard-horizontal.component.html

```diff
@@ -13,16 +13,19 @@
             [attr.aria-label]="step.ariaLabel || null"
             [attr.aria-labelledby]="(!step.ariaLabel && step.ariaLabelledby) ? step.ariaLabelledby : null"
             [index]="i"
-            [state]="_getIndicatorType(i, step.state)"
+            [state]="getStepState(i, step.state)"
             [label]="step.stepLabel || step.label"
             [selected]="selectedIndex === i"
             [active]="step.completed || selectedIndex === i || !linear"
+            [completed]="step.completed"
             [optional]="step.optional"
             [errorMessage]="step.errorMessage"
-            [disableRipple]="disableRipple">
+            [disableRipple]="disableRipple"
+            [stepIcon]="step.icon"
+            [customIcons]="customIcons">
         </wizard-step-header>
-        <div *ngIf="!isLast" class="nui-wizard-horizontal-line"></div>
-    </ng-container>
+        <div *ngIf="!isLast" class="nui-wizard-horizontal-line" [ngClass]="{'nui-wizard-horizontal-line-completed': step.completed}"></div>
```

Comment:
I think a proper BEM class name would be something like the following. In addition to the underscores indicating the "element" part of BEM, I've added a name change suggestion. "connector" since it makes it more obvious when looking at the code what it's for, and removed the "completed" since the implementation detail for when to show it doesn't need to be embedded in the name.

```suggestion
        <div *ngIf="!isLast" class="nui-wizard-horizontal-line" [ngClass]="{'nui-wizard-horizontal__step-connector-line': step.completed}"></div>
```